scanpipe/api/views.py

```diff
@@ -168,6 +168,8 @@ def results_download(self, request, *args, **kwargs):
             output_file = output.to_cyclonedx(project, **output_kwargs)
         elif format == "attribution":
             output_file = output.to_attribution(project)
+        elif format == "ort":
```

Comment:
@tdruez  I propose we don't call this format just `ort` but something like `ort-package-list` as ORT current support 2 input format and maybe more in the future. Like make things future proof.